targets/x86-geode/profiles.mk

```diff
@@ -0,0 +1,8 @@
+X86_GENERIC_NETWORK_MODULES := kmod-3c59x kmod-e100 kmod-natsemi kmod-ne2k-pci kmod-pcnet32 kmod-8139too kmod-r8169 kmod-sis900 kmod-tg3 kmod-via-rhine kmod-via-velocity kmod-forcedeth
```

Comment:
This list has been extended in the other x86 targets, do you think it would make sense to add the other modules in x86-geode as well?

Also, to keep this consistent with the other targets, the variable should be renamed to `X86_GEODE_NETWORK_MODULES`.